tools/ingest-converter/src/test/resources/org/logstash/ingest/logstashComplexCase4.conf

```diff
@@ -0,0 +1,31 @@
+filter {
+   grok {
+      match => {
+         "message" => "%{COMBINEDAPACHELOG}"
+      }
+   }
+   if "_grokparsefailure" in [tags] {
+         mutate {
```

Comment:
@suyograo the indentation is kinda messed up here, isn't it? this should only be indented by 3 spaces relative to the line below. Now we're closing this one in line 12 below, which looks wrong, doesn't it?